.github/workflows/stale_labeler.yaml

```diff
@@ -0,0 +1,19 @@
+name: 'Label inactive PRs as stale'
+
+on:
+  schedule:
+    - cron: '0 11 * * *'  # runs at 11am UTC every day => early morning in US
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8  # https://github.com/marketplace/actions/close-stale-issues
+        with:
+          stale-pr-label: 'stale'
+          stale-pr-message: "This PR is being marked as stale because there hasn't been activity in 14 days. It may be closed by a rosdistro maintainer. If this label isn't appropriate, you can remove the label and add the \"don't mark as stale\" label. You can find our community contributing guidelines here: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md."
```

Comment:
One thing about this message is that users will typically not have the permissions to manipulate labels on rosdistro PRs.  So that wording should be modified to "ask a maintainer to label it".  Also, I'd prefer if the label were a bit shorter, and didn't have a negation in it; I like @tylerjw's suggestion of "persistent", but I'm open to other suggestions as well.

Reply:
Done in 0e782a8.